### DISCUSSION TITLE
千萬不要錯過，將會徹底改變世界的元宇宙 | 老高與小茉 Mr & Mrs Gao

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#214] [General]
https://www.youtube.com/watch?v=hm5K-PBz0rg
真的超喜歡老高ㄟ，我是老高的忠實粉絲

**Comment by handsomeniko** [comment]
老高確實滿有趣的，但有些就當故事看看就好，有些知識他有講錯過(例如:重力異常的解釋)

**Comment by Shiba010** [comment]
老高的影片真的不錯，可以配飯享用。

**Comment by ayakasanma** [comment]
老高真的是流量王者，想跟他拜師掌握財富密碼😂

**Comment by ghost** [comment]
謝謝你分享老高的影片 多認識一位YTberXD
